### DISCUSSION TITLE
Predicting Kps

### DISCUSSION THREAD

**Discussion by pburkat** [Open-Systems-Pharmacology/Forum#1161] [Questions & Problems]
Hello, is there any information available on the differences between the PK-Sim standard vs. Rodgers Rowland calculations for partition coefficients?

**Comment by prvmalik** [comment]
This paper will give the best overview https://dmd.aspetjournals.org/content/48/10/903.abstract

General method to determine best fit between partition coefficient algorithms is to compare model performance for curve shape when lipophilicity is fitted against an IV profile.

From experience, try R-R first if dealing with a weak acid or weak base compound in humans.

If neutral compound or tightly plasma-protein bound, I may suggest that it will be more likely that PK-Sim standard will be a better fit for curve shape.
